turbopack/crates/turbopack-ecmascript/src/references/pattern_mapping.rs

```diff
@@ -83,7 +94,7 @@ pub(crate) enum PatternMapping {
     /// ```js
     /// require(`./images/${name}.png`)
     /// ```
-    Map(FxIndexMap<String, SinglePatternMapping>),
+    Map(#[bincode(with = "turbo_bincode::indexmap")] FxIndexMap<String, SinglePatternMapping>),
```

Comment:
This annotation will have to be specified for every occurrence of indexmap?

Reply:
But you do get an error message when forgetting this? Or does it silently use the wrong serialization impl

It's fine, not much we can do apart from trying to upstream.